src/_lib/modules/charts/tooltip/tooltip.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { ChartTooltipDirective } from './chart-tooltip.directive';
```

Comment:
This filename for this spec should update based on the update to the directive filename, and these imports will need to change. I'd say some useful tests could be written here too.